pkg/ui/src/views/cluster/containers/clusterOverview/cluster.styl

```diff
@@ -31,6 +31,17 @@
     grid-template-rows repeat(3, auto)
     grid-template-areas "cap-t cap-t cap-t cap-t . live-t live-t live-t . rep-t rep-t rep-t" "cap-m cap-c cap-c cap-c . live-a live-b live-c . rep-a rep-b rep-c" "cap-a1 cap-a2 cap-a3 cap-a4 . live-1 live-2 live-3 . rep-1 rep-2 rep-3"
 
+    @media screen and (max-width: 960px)
+      grid-template-columns 6fr 8fr 6fr 8fr
+      grid-template-rows repeat(3, auto) 16px repeat(3, auto) 16px repeat(3, auto)
+      grid-template-areas "cap-t cap-t cap-t cap-t" "cap-m cap-c cap-c cap-c" "cap-a1 cap-a2 cap-a3 cap-a4" ". . . ." "live-t live-t live-t ." "live-a live-b live-c ." "live-1 live-2 live-3 ." ". . . ." "rep-t rep-t rep-t ." "rep-a rep-b rep-c ." "rep-1 rep-2 rep-3 ."
```

Comment:
whoah, didn't see these when you first implemented this. nothing you can do since it's just the CSS spec, but not the easiest to read!

Reply:
ugh stylus ruins this.  it looks much nicer in plain css (ok, still weird, but at least you can multi-line it)